podcast/2.19.md

```diff
@@ -0,0 +1,745 @@
+
+So even if people will decide to move from Cirrus, there is no need to convert existing Cirrus configuration into some other CI system, because Cirrus CLI can run locally.
+
+**Beyang Liu:**
+
+Oh, wow. So I can use this to run my build locally?
+
+**Fedor Korotkov:**
+
+Yes.
+
+**Beyang Liu:**
+
+I feel like that's useful even if I'm not concerned about the switching thing. Because there's so many times where the thing failed in the build environment, but when I run the test locally, it seems to succeed. Now you're saying you could use Cirrus CLI to reproduce the CI environment on your local machine.
+
+**Fedor Korotkov:**
+
+Yeah. Only Docker images, and this …
+
+**Beyang Liu:**
+
+Yeah, yeah, of course.
+
+**Fedor Korotkov:**
+
+… and the most popular instances that Cirrus runs. That's why we were very late with SSH access, because you always could use Cirrus CLI to run your CI locally to reproduce things. And 99.9% of the cases, you will be able to reproduce it.
+
+But then there is, of course, problems when you don't have, for example, FreeBSD access, or you don't have macOS. And your tests are only failing on one particular virtual machine, for example. Then it's more convenient to just rerun things in the browser you have. With the terminal access in the browser, you can figure things out and then fix it.
+
+**Beyang Liu:**
+
+Cool. Anything else that you want to call out? We're coming up on the end of the hour here. So are there any final calls to action, or things you want to highlight about Cirrus, for people listening?
+
+**Fedor Korotkov:**
+
+No, I was just really glad to join you on the podcast, and tell you and all the listeners more about Cirrus. I'm very excited. I'm biased, of course. But I think there were very interesting design choices, and just overall architectural decisions we made to make Cirrus very unique. And it will be great if some people will check it out. Maybe they will share my excitement about it.
+
+**Beyang Liu:**
+
+If they want to try it out, maybe install it on one of their open-source repositories, how do they go about doing that?
+
+**Fedor Korotkov:**
+
+They can just Google Cirrus CI, and it's as easy as installing a Cirrus CI application from GitHub marketplace. And we have a very nice integration with GitHub. We support annotation. So all the failing things will be highlighted right on the PR.
+
+We support fancy GitHub Checks API, where you … in most cases, you don't even need to go to Cirrus UI, because everything is available on your pull request, or UI inside GitHub. So, yeah, it's super easy. And then we have a lot of examples of how to use, and we support, as we discussed, a lot of use cases and scenarios to test things.
+
+**Beyang Liu:**
+
+Awesome. Well, thanks so much, Fedor, for taking the time to be on the show. Real pleasure chatting with you.
+
+**Fedor Korotkov:**
+
+Thank you, Beyang.
+
```

Comment:
@nickmyyz @csbailey5t I noticed we did not include the "This transcript has been lightly edited for clarity and readability." sentence at the end. Can you please add this?